openshift_sti_images/ruby.adoc

```diff
@@ -111,10 +110,25 @@ By omitting the `VERSION` parameter, the build/test action will be performed on
 * `hack/` - Folder contains scripts which are responsible for build and test actions performed by the `Makefile`.
 * `Makefile` - Make utility for simplifying image build and test actions.
 
+== Image name structure
+
+*Structure: openshift/`1`-`2`-`3`*
```

Comment:
I would change this to:

    *Structure: `openshift/<platform_name>-<platform_version>-<base_builder_image>`*  
    
    * platform_name - Refers to the STI platform; `ruby` is one example
    * platform_version - The version of the referenced platform, without dots (like `20` for Ruby 2.0)
    * base_builder_image - The base OS, like `rhel7` or `centos7`